sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -144,7 +144,7 @@ class LegacyFastTimestampFormatter(
     cal.getTimeInMillis * MICROS_PER_MILLIS + micros
   }
 
-  def format(timestamp: SQLTimestamp): String = {
+  def format(timestamp: Long): String = {
```

Comment:
`microsec`?

Reply:
Renamed to `micros` for consistency with other places.

Reply:
nit: it still looks `timestamp`.